app/src/components/tables/DeploysTable/DeploysTable.tsx

```diff
@@ -39,35 +45,127 @@ export const DeploysTable: React.FC = () => {
   const header = useMemo(() => <div>header placeholder</div>, []);
   const footer = useMemo(() => <div>footer placeholder</div>, []);
 
-  const columns = useMemo<ColumnDef<ApiData.SidecarDeploy>[]>(
+  const columns = useMemo<ColumnDef<ApiData.ProcessedSidecarDeploy>[]>(
     () => [
       {
         header: `${t('deploy-hash')}`,
-        id: 'deploy_hash',
-        accessorKey: 'deploy_hash',
-        cell: ({ getValue }) => getValue<string>(),
+        id: 'deployHash',
+        accessorKey: 'deployHash',
+        cell: ({ getValue }) => (
+          <HashAndCopyToClipboardWrapper>
+            <StyledHashLink
+              to={{
+                pathname: `/deploy/${getValue<string>()}`,
+              }}>
+              {truncateHash(getValue<string>())}
+            </StyledHashLink>
+            <StyledCopyToClipboard textToCopy={getValue<string>()} />
+          </HashAndCopyToClipboardWrapper>
+        ),
         enableSorting: false,
       },
       {
         header: `${t('block-hash')}`,
-        id: 'block_hash',
-        accessorKey: 'deploy_processed.block_hash',
-        cell: ({ getValue }) => getValue<string>(),
+        id: 'blockHash',
+        accessorKey: 'blockHash',
+        cell: ({ getValue }) => (
+          <HashAndCopyToClipboardWrapper>
+            <StyledHashLink
+              to={{
+                pathname: `/block/${getValue<string>()}`,
+              }}>
+              {truncateHash(getValue<string>())}
+            </StyledHashLink>
+            <StyledCopyToClipboard textToCopy={getValue<string>()} />
+          </HashAndCopyToClipboardWrapper>
+        ),
         enableSorting: false,
       },
+      {
+        header: `${t('public-key')}`,
+        id: 'publicKey',
+        accessorKey: 'publicKey',
+        cell: ({ getValue }) => (
+          <HashAndCopyToClipboardWrapper>
+            <StyledHashLink
+              to={{
+                pathname: `/account/${getValue<string>()}`,
+              }}>
+              {truncateHash(getValue<string>())}
+            </StyledHashLink>
+            <StyledCopyToClipboard textToCopy={getValue<string>()} />
+          </HashAndCopyToClipboardWrapper>
+        ),
+        enableSorting: false,
+      },
+      {
+        header: `${t('age')}`,
+        accessorKey: 'timestamp',
+        cell: ({ getValue }) => (
+          <Age>{formatTimeAgo(new Date(getValue<number>()))}</Age>
+        ),
+        enableSorting: false,
+      },
+      {
+        header: `${t('contract')}`,
+        accessorKey: 'contractType',
+        cell: ({ getValue }) => capitalizeWords(getValue<string>()),
```

Comment:
There is a way to do this using CSS `text-transform: capitalize;`

I understand this is the value setter so it's probably not worth using here

Reply:
Ah cool, I didn't know that existed. And this would be able to capitalize `All Words In A Sentence` or `Just the first`?